.github/workflows/deploy.yml

```diff
@@ -27,17 +27,33 @@ jobs:
   deploy:
     name: Deploy
     runs-on: ubuntu-latest
+    container:
+        image: ubuntu:latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: ruby/setup-ruby@v1
+      - name: Update packages
+        run: |
+            apt-get update -qq
+      - name: Install dependencies
+        run: |
+            apt-get install -qq -y jq rsync git libyaml-0-2 npm
+      - name: Checkout git repository
+        uses: actions/checkout@v2
+      - name: Set GitHub workspace as git safe.directory # Required to work around: https://github.com/actions/checkout/issues/766
+        run: |
+            git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
I think in `actions/checkout@v3` the `set-safe-directory` default `true` was added to not require this.
It can also be overriden to set the path: https://github.com/actions/checkout#usage
like:
```
      - name: Checkout git repository
        uses: actions/checkout@v3
        with:
            set-safe-directory: "$GITHUB_WORKSPACE"
```
but I think the default is adding it to safe-directory already.

Reply:
Ah, I didn't realize this changed in v3. Thanks, @raulcd! I'll see if this works fine with `checkout@v3` on the `mathworks/arrow-site` fork.

Reply:
It's possible I am doing something wrong, but this doesn't seem to be working: https://github.com/mathworks/arrow-site/actions/runs/4314046395/jobs/7526593648.

Reply:
@kevingurney does it work with just?
```
      - name: Checkout git repository
        uses: actions/checkout@v3
```
I'll investigate the correct usage in the meantime

Reply:
I think the change you are referring to is actions/checkout#762?

This quote below makes me think that I still need to set `safe.directory` separately from the checkout since the original setting doesn't persist across workflow steps:

> This does carry some limitations, it doesn't persist this configuration for the duration of the job mainly. So if your job pushes to git or something after checkout, that will continue to fail. We need to figure out how to address this at an ecosystem level, outside of the checkout action.

It's possible I am mistaken, though.

Reply:
Sorry, for some reason I didn't see you previous comment before positing my most recent one.

> @kevingurney does it work with just?

      - name: Checkout git repository
        uses: actions/checkout@v3


Unfortunately, no, that doesn't seem to work. See: https://github.com/mathworks/arrow-site/actions/runs/4314015945/jobs/7526526833

Reply:
No worries at all! :) Agreed - updating to `actions/checkout@v3` seems like a good idea. I'll push a commit to make that change now.